.tekton/integration-tests/tasks/rhtap-e2e.yaml

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
This task should install rhtap not creating another task for e2e.

Reply:
@flacatus okay, so it should refer to this e2e [task](https://github.com/konflux-qe-incubator/konflux-qe-definitions/blob/main/integration-test/rhtap/rhtap-e2e/task/rhtap-e2e-runner.yaml) , right?

Reply:
right!